themes/Pale Fire-color-theme.json

```diff
@@ -236,23 +254,23 @@
 			"settings": {
 				"fontStyle": "underline"
 			}
```

Comment:
Oops, looks like VS Code autoformat fixed the indentation here again, which will cause a conflict with #4. I’ll remove it.

Reply:
And it seems I selected the wrong part of the file to comment on 🤦.

Reply:
I’ve restored the original indentation.